test/integration/Uart/uart_read.c

```diff
@@ -0,0 +1,85 @@
+/**@file
+ * This file is part of the Leon3 BSP for the Test Environment.
```

Comment:
Where did we get this copyright from? I think it might now be accurate for this particular project.  I think that the repository license is GPL 2.0. 
As for the "financed by" statement, we will fix it at the end of the project, we will need to confirm the proper wording with SENER.

Reply:
ack